modules/EmergencyMedicine/sql/0_1_0-to-0_2_0_upgrade.sql

```diff
@@ -293,3 +293,13 @@ INSERT INTO `fhir_value_set_systems` (`vs_id`, `system`, `type`,`filter`)
 VALUES ('reason_codes_1', 'clinikal_reason_codes', 'Filter', '1');
 #EndIf
 
+
+#IfNotRow2D list_options list_id lists option_id clinikal_app_secondary_statuses
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`, `edit_options`)
```

Comment:
I think we need to put empty list in the fhirApi module and only the values here, I afraid it's can cause bugs if somebody does not install the vertical repository, I think it makes more sens

Reply:
we have like 10 ton of sql dependencies between the verticals 
but I dont mine put this list in FHIR API